src/hooks/useReadingFluencyAsset.test.ts

```
@@ -0,0 +1,70 @@
+      'https://cdn.example.com/success.gif';
+
+    const { result } = renderHook(() =>
+      useReadingFluencyAsset({ apiClient, institutionId })
+    );
+
+    expect(result.current).toBe('https://cdn.example.com/success.gif');
+  });
+
+  it('returns the consumer-provided fallback when the institution asset is missing', () => {
+    const { result } = renderHook(() =>
+      useReadingFluencyAsset({ apiClient, institutionId, fallback: '/local.gif' })
```

ESLint

[error] Replace `·apiClient,·institutionId,·fallback:·'/local.gif'` with `⏎········apiClient,⏎········institutionId,⏎········fallback:·'/local.gif',⏎·····`

(prettier/prettier)